test/llilc_checkpass.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
+#
+#title           :llilc_checkpass.py
+#description     :
+# 
+# llilc_checkpass compares two test rusuts, no matter summary or verbose,
+# checks if any function successfully jitted by LLILC in base result
+# failed jitting in diff result.
+#
+# If the check passed, the script return 0. In case of 0, it might have
+# jitted more functions in diff result than in base result. The two results
+# are not necessary the same in this case. Counting result is reported.
+# 
+# If the check failed due to newly faily routine, return 1.
+#     
+# If the check failed due to unexpected reason, return negative numbers.
+#
+# usage: llilc_checkpass.py [-h] -b BASE_RESULT_PATH -d DIFF_RESULT_PATH
+#
+# optional arguments:
+#   -h, --help            show this help message and exit
+#   -b BASE_RESULT_PATH, --base-result-path BASE_RESULT_PATH
+#                         full path to base result
+#   -d DIFF_RESULT_PATH, --diff-result-path DIFF_RESULT_PATH
+#                         full path to diff result
+#==========================================================================================
+
+import argparse
+import re
+import sys
+import os
+import difflib
+
+def main(argv):
+    # Parse the command line    
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-b", "--base-result-path", type=str, required=True,
+                        help="full path to base result")
+    parser.add_argument("-d", "--diff-result-path", type=str, required=True,
+                        help="full path to diff result")
+    args, unknown = parser.parse_known_args(argv)
+
+    if unknown:
+        print("Unknown argument(s): ", ", ".join(unknown))
+        return -2
```

Comment:
Might want to use symbolic constants for the various return codes, so they're a bit more meaningful.